client/src/components/PosterList.js

```diff
@@ -0,0 +1,47 @@
+import PropTypes from 'prop-types'
+import styled from 'styled-components/macro'
+import Poster from './Poster'
+
+PosterList.propTypes = {
+  list: PropTypes.array,
+}
+
+export default function PosterList({ list }) {
+  return (
+    <StyledPosterList>
+      {list.map(({ id, poster_path, name }) => {
```

Comment:
Many developers prefer to rename something like `poster_path` to `posterPath` as soon as possible after getting the data.